packages/okta-angular/README.md

```diff
@@ -161,7 +183,7 @@ const appRoutes: Routes = [
 
 ### `OktaLoginRedirectComponent`
 
-By default, the `OktaLoginRedirect` component redirects users to your Okta organization for login. Simply import and add it to your `appRoutes` to offset authentication to Okta entirely:
+The `OktaLoginRedirect` component redirects the user's browser to the Okta-hosted login page for your organization. For more advanced cases, this component can be copied to your own source tree and modified as needed.
```

Comment:
Do we have any examples for this "more advanced case"?  If not that shouldn't hold up this change, but sounds like a good idea of a guide/etc to add and link to from here.

Reply:
They may want to add a delay w/ an animation, or perhaps do some additional checks before login. Maybe drop a cookie of their own before the redirect for extra validation on the callback. The main idea I want to communicate is they are not forced to use this component. What it does is very simple, but they can copy it if they want something more complex.